src/applications/vaos/actions/confirmed.json

```diff
@@ -1,28 +1,900 @@
-[{
-  "id": "123",
-  "facilityId": "888",
-  "facility": {
-    "id": "888",
-    "name": "Washington VA Medical Center",
-    "type": "VAMC",
-    "address": "50 Irving St NW",
-    "city": "Washington",
-    "state": "DC",
-    "directScheduleEnabled": true,
-    "parentSiteCode": 688
+{
+  "lastAccessDate": "09/04/2019 20:52:01",
```

Comment:
I think we'll need a later ticket to update this with the data from the appointments and booked-cc-appointments service. I couldn't get that yesterday because CAG (and VAOS) were down a lot.